### DISCUSSION TITLE
Random charges on my credit card and support ticket is not moving.

### DISCUSSION THREAD

**Discussion by nikushx** [community/community#112503] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I need to figure out why I have a random $20 charge on a card I've never linked to GitHub. My support ticket has been sitting unanswered.

**Comment by ettaboyle** [comment]
Hi @nikushx! 

As you've already found, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). We cannot respond to or help with any account specific questions in this public forum. I am closing and locking this discussion. 

Please do not create discussions about account specific questions in the future. Thank you!
